tests/ui/parallel-rustc/deadlock-issue-119785.rs

```diff
@@ -0,0 +1,74 @@
+//@ parallel-front-end
+//@ compile-flags: -Z threads=200
+
+#![allow(incomplete_features)]
+#![feature(
+    const_trait_impl,
+    effects,
+)]
+
+use std::marker::Destruct;
+
+const fn cmp(a: &impl ~const PartialEq) -> bool {
+    //~^ ERROR
```

Comment:
Question: is the exact content and order of these errors important?

Reply:
No it isn't, thanks :) I just deleted all the error context